pydarn/utils/superdarn_radars.py

```diff
@@ -1,18 +1,18 @@
 # Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
```

Comment:
Amend this file if hdw master branch changes to main before release.

Reply:
I've merged the hdw PR and renamed the default branch to `main`

Reply:
Great! I'll give it a quick test and start with the release this afternoon. I guess if I come up against any big problems I'll give you a call tomorrow morning as it must be getting late for you now!